src/status_im2/contexts/onboarding/common/style.cljs

```diff
@@ -0,0 +1,37 @@
+(ns status-im2.contexts.onboarding.common.style
+  (:require
+    [quo2.foundations.colors :as colors]))
+
+(def default-margin 20)
+
+(defn page-container
+  [insets]
+  {:flex             1
+   :padding-top      (:top insets)
+   :background-color colors/neutral-80-opa-80-blur})
+
+(def title-container
+  {:justify-content    :center
+   :margin-top         12
+   :padding-horizontal default-margin})
+
+(def title-text
```

Comment:
hmm, I believe we don't want to add common stylings that are used in multiple views. It would be better to be created as common components instead since they all use the same configuration.

```
(ns status-im2.contexts.onboarding.common.components

(defn title [{:keys [accessibility-label] }]
 [quo/text
    {:accessibility-label accessibility-label
     :weight              :semi-bold
     :size                :heading-1
     :style               onboarding-style/title-text})
    ```

Reply:
but tbh, in the case of title component it should actually be created in quo2 as there is this standard title component:
https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?node-id=8824-151323&t=TiQujkTRkiSuEyu7-0